README.md

```diff
@@ -134,6 +134,11 @@ Type: `Array`
 Default value: `["http","https"]`  
 Will only check links with schemes/protocols mentioned in this list. Any others (except those in `excludedSchemes`) will output an "Invalid URL" error.
 
+### options.cacheResponses
+Type: 'Boolean'
+Default Value: 'false'
```

Comment:
Also needs two spaces before the line break as this produces a margin-less `<p>`in github's markdown->html compiler.